charts/kueue/values.yaml

```diff
@@ -23,7 +23,7 @@ controllerManager:
       pullPolicy: IfNotPresent
   manager:
     image:
-      repository: us-central1-docker.pkg.dev/k8s-staging-images/kueue/kueue
+      repository: registry.k8s.io/kueue/kueue
```

Comment:
what is this replacement reason?

Reply:
I'm not sure. I will check in the next 2-3h

Reply:
My guess would be that for a release, the install should not pull from staging-images.

Reply:
> My guess would be that for a release, the install should not pull from staging-images.

This is for main branch, that indicates development image, IMO.

Reply:
I suspect this was some non-committed change from previous steps, but not sure. I repeated the steps and it didn't generate. PTAL.

Reply:
/lgtm